### PR TITLE
fix explanation of Points vs Scatter

### PR DESCRIPTION
**Pull request by ltalirz**
It is the Scatter element where y is considered dependent on x, not the
Points element.

### CODE REVIEW

**Comment by philippjfr** [comment]
Thanks! I'm actually about to push a PR which explicitly disallows a second key dimension on Scatter (which should have been done ages ago). In the long term the semantic differences will become clearer, e.g. eventually a second key dimension on a Scatter element will result in a two-level categorical axis unlike Points were it simply declares the two dimensional space.

**Comment by philippjfr** [comment]
Thanks again!

**Comment by ltalirz** [comment]
You're welcome! Thanks for this great package!

File: examples/reference/elements/bokeh/Points.ipynb [hunk 1]
```diff
@@ -97,7 +97,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "**Note**: Although the  ``Scatter`` element is superficially similar to the [``Points``](./Points.ipynb) element (they can generate plots that look identical), the two element types are semantically quite different. The fundamental difference is that [``Points``](./Points.ipynb) are used to visualize data where the *y* variable is *dependent*. This semantic difference also explains why the histogram generated by ``hist`` call above visualizes the distribution of a different dimension than it does for [``Scatter``](./Scatter.ipynb).\n",
+    "**Note**: Although the  ``Scatter`` element is superficially similar to the [``Points``](./Points.ipynb) element (they can generate plots that look identical), the two element types are semantically quite different. The fundamental difference is that [Scatter](./Scatter.ipynb) is used to visualize data where the *y* variable is *dependent*. This semantic difference also explains why the histogram generated by ``hist`` call above visualizes the distribution of a different dimension than it does for [``Scatter``](./Scatter.ipynb).\n",
```
Comment (jbednar) [inline examples/reference/elements/bokeh/Points.ipynb]: Oops! If I wrote the original, I apologize profusely for confusing generations of users.

**Pull request merged**